### ISSUE TITLE
Lutris can't run on ArchLinux,

### ISSUE THREAD

**Issue by sekaom** [lutris/lutris#4754]
### Bug description

I can't run lutris on my computer,it's installed Archlinux and python2 and 3.but it has some errors,like its log output

### How to Reproduce

maybe  
lutris

### Expected behavior

Works normally and starts

### Log output

```shell
╰─ lutris -d
Traceback (most recent call last):
  File "/usr/bin/lutris", line 52, in <module>
    from lutris.gui.application import Application  # pylint: disable=no-name-in-module
  File "/usr/lib/python3.10/site-packages/lutris/gui/application.py", line 29, in <module>
    gi.require_version("Gdk", "3.0")
AttributeError: module 'gi' has no attribute 'require_version'
```

### System Information

```shell
null ,because i can't run it
```

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by legluondunet** [comment]
try to install python3-gi package

**Comment by sekaom** [comment]
I deleted all the files under/usr/lib/python3.10/site-packages/and reinstalled python and lutris, despite being prompted to be missing some modules, I followed the prompts to install these modules using pip. Now, lutris is ready to run.
I run it in terminus and it shows that some files cannot be downloaded. Opening the link shows 404, but I can open thehttps://raw.githubusercontent.com/lutris/lutris-runtime/master/icons/hicolor/symbolic/apps/vice-symbolic.svgnormally. For example, thishttps://raw.githubusercontent.com/lutris/lutris-runtime/master/icons/hicolor/symbolic/apps/reicast-symbolic.svg:. It seems that there is an ":" after the link.
Now I'm waiting for the download to complete
